proto/metadata.proto

```diff
@@ -491,6 +491,15 @@ message Metadata {
     bool set_no_peer_group = 175;
     // Bgp community member is a string
     bool bgp_community_member_is_a_string = 176;
+    // support for ipv4 to ipv6 next hop
+    bool ipv4_static_route_with_ipv6_nh_unsupported = 177;
+    // support for ipv6 to ipv4 next hop
+    bool ipv6_static_route_with_ipv4_nh_unsupported = 178;
+    // support for static route with drop nh
+    bool static_route_with_drop_nh = 179;
+    // support for static route with drop nh
```

Comment:
this comment isn't correct, (copy pasted from the previous one)

otherwise lgtm

Reply:
Revised comments

Reply:
@nandanarista I have resolved the comments let me know if there are any issues with these comments. If so can you add more specifics on what you may be looking for. and if no further issues please approve and merge if no further changes requested

Reply:
lgtm, i've approved this, but it doesn't count as an approval required for merge, you'll need a maintainer to approve.